### PR TITLE
chore(docs): add/update info

### LINKED ISSUE #201: docs:  cosmetic cleanup and content
The following cosmetic cleanups need to happen:
- [x] GitHub button link
- [x] TOC Links to components
- [ ] Additional content for Getting Started Section
 - [ ] Update dependencies (Tether?  Etc...)
 - [ ] Installation instructions
   - [ ] npm
    - [ ] anything else?
 - [ ] Usage/Import instructions
 - [x] Build Instructions
   - [x] Link to DEVELOPER
 - [x] Contribution Instructions
  - [ ] Link to README
   - [x] Link to CONTRIBUTING
 - [x] Code of Conduct
  - [x] Link to CODE_OF_CONDUCT
- [x] GitHub Issue Template
- [x] GitHub PR Template

### PR DESCRIPTION
**Pull request by icfantv**
* add issue template
* add PR template
* update README
* add more info to Getting Started

fixes #201

### CODE REVIEW

**Comment by icfantv** [comment]
A lot of this was pulled over from the UIBS project and tweaked.  I suspect it will require more tweaking.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Before submitting a pull request, please make sure you have at least performed the following:
+
+ - [] read and followed the [CONTRIBUTING.md](https://github.com/ng-bootstrap/core/blob/master/CONTRIBUTING.md) guide.
+ - [] built and tested the changes locally.
+ - [] added any applicable tests.
```
Comment (pkozlowski-opensource) [inline .github/PULL_REQUEST_TEMPLATE.md]: Shell we add something about updating docs and demos?

**Comment by pkozlowski-opensource** [comment]
LGTM with one minor comment.

**Comment by icfantv** [comment]
@pkozlowski-opensource updated per comment.

**Pull request closed without merging**